djmoney/contrib/django_rest_framework/fields.py

```diff
@@ -12,6 +14,18 @@ class MoneyField(DecimalField):
     does decimal's validation during transformation to native value.
     """
 
+    def __init__(self, *args, **kwargs):
+        # patches wrong validators for drf
+        super().__init__(*args, **kwargs)
+        validators = self.validators
+        print(type(validators))
+        for i in range(len(validators)):
+            if isinstance(validators[i], MinValueValidator):
+                validators[i] = MinMoneyValidator(self.min_value)
+            elif isinstance(validators[i], MaxValueValidator):
+                validators[i] = MaxMoneyValidator(self.max_value)
+        self.validators = validators
```

Comment:
Since `validators` is pointing to `self.validators` it shouldn't be required because the list is modified  during the `for` loop above